hack/reduce-snapshot.sh

```diff
@@ -65,4 +76,4 @@ if [ "${SINGLE_COMPONENT}" == "true" ]; then
 fi
 
 # we need to create snapshot file to be passed to later stages.
-echo "${SNAPSHOT}" | jq '.' | tee "${SNAPSHOT_PATH}"
+cp "$WORKING_SNAPSHOT" "$SNAPSHOT_PATH"
```

Comment:
Why remove the tee? I think it can be kept safely like this:

```suggestion
jq '.' "${WORKING_SNAPSHOT}" | tee "${SNAPSHOT_PATH}"
```

Reply:
I just didn't know what value any of it added. If there's value in it, I can definitely add it back.

Reply:
I presume the idea was to make it visible in the taskRun log.

Reply:
We could reconsider if that's useful, but I think it's better not to change it in this PR.

Reply:
I guess it would fit better with the debug output, since the component list is clearly shown in the next step. (I'm looking in https://console.redhat.com/application-pipeline/workspaces/rhtap-contract/applications/ec-main-ci/pipelineruns/ec-main-ci-enterprise-contract-9jfvx ).

(But still, better to change that it a later PR.)